pre_commit/simple_deactivatable_regexp_checker.rb

```diff
@@ -19,8 +21,9 @@ def errors?
 
   def examine_code
     return [] unless use_for_project?
+    return [] unless check_file_based_on_extension?
     mess = []
-    if !PreCommitHelper.directory_excluded_from_checks?(@dir)
+    if !PreCommitHelper.directory_excluded_from_all_checks?(@dir)
```

Comment:
Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.